src/types/FilterType.ts

```diff
@@ -0,0 +1,5 @@
+export enum FilterType {
+  ALL = 'all',
+  ACTIVE = 'active',
+  COMPLETE = 'complete',
```

Comment:
The naming of the enum value should be consistent with others. It's better to use 'COMPLETED' instead of 'COMPLETE' 📘